sdk/typescript/scripts/generate.ts

```diff
@@ -173,6 +174,9 @@ const options: {
 					alias: 'signature',
 					typeAlias: 'string | string[]',
 				},
+				requestType: {
+					deprecated: 'requestType will be ignored by JSON RPC',
```

Comment:
Not sure this is the right message right now, since it won't be ignored.  Probably should say something like "will be ignored by JSON RPC in the future"